addons/website/static/src/snippets/s_embed_code/000.js

```diff
@@ -0,0 +1,41 @@
+/** @odoo-module **/
+
+import publicWidget from "web.public.widget";
+import { _t } from "@web/core/l10n/translation";
+
+const EmbedCodeWidget = publicWidget.Widget.extend({
+    selector: ".s_embed_code",
+    disabledInEditableMode: false,
+
+    /**
+     * @override
+     */
+    async start() {
+        this.embedCodeEl = this.el.querySelector(".s_embed_code_embedded");
+        if (this.editableMode && this.embedCodeEl.offsetHeight === 0) {
+            this._addPlaceholder();
+        }
+        return await this._super(...arguments);
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        this._super(...arguments);
+        const embedContent = this.embedCodeEl.dataset.embedContent;
+        if (!this.editableMode && embedContent && embedContent.match(/<script[^<]*?>/)) {
```

Comment:
Regex seems strange, what's the point of `[^<]`? Also this probably matches `<scriptstuff>` (weird case I agree). Maybe it's easier to match to closing tag rather than the opening one? You could just search for `</script>` (to see if weird people can put some spaces in there though?).

Also use `test` instead of `match` if you just want to test.